README.md

```diff
@@ -1 +1,200 @@
-# Final-Group-Capstone-Backend
\ No newline at end of file
+# 📗 Table of Contents
```

Comment:
_note: There is no open PR in the frontend repo for me to review.  Therefore, I'll include the review comments here._

# Frontend review

- [ ] On the mobile view, I couldn't find a way to log out.  If it's already there in the menu, kindly make it more obvious.  If it's not, kindly include it so users can log out on their phones.  👍 

- [ ]  Please, kindly refactor the functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://github.com/billymohajeri/Final-Group-Capstone-Backend/assets/57421823/494a8cd3-e424-4013-b9bb-d5808af4731e)


That way, your code will meet the latest standards. 💪 

- [ ] Since you have no frontend tests, kindly remove the `App.test.js` and `setupTests.js` that came with React if not using them.

Reply:
Done, thanks 👍